.github/workflows/ci.yaml

```diff
@@ -8,12 +8,6 @@ on:
     branches:
       - main
   workflow_dispatch:
-    inputs:
-      no-cache:
-        description: "Run tests without cache"
-        default: false
-        required: false
-        type: boolean
```

Comment:
This is unrelated to the PR, but I am removing cache option.
It has no effect at all.